src/sdb.h

```diff
@@ -206,6 +206,12 @@ SDB_API bool sdb_disk_insert(Sdb* s, const char *key, const char *val);
 SDB_API bool sdb_disk_finish(Sdb* s);
 SDB_API bool sdb_disk_unlink(Sdb* s);
 
+/* plaintext sdb files */
+SDB_API bool sdb_text_fsave(Sdb *s, FILE *f, bool sort);
```

Comment:
FILE is not used in SDB until now, if we have the same callbacks for bin and text we can easily wrap them into a generic struct with callbacks to select different backends